### DISCUSSION TITLE
Codespaces usage disappearing with none created

### DISCUSSION THREAD

**Discussion by QuartzWarrior** [community/community#63530] [Codespaces]
![Screenshot_20230811_153548_Brave.jpg](https://github.com/community/community/assets/74436633/2f953606-997e-4452-b2a9-eeff31077fdb)

![Screenshot_20230811_153559_Gmail.jpg](https://github.com/community/community/assets/74436633/b5ab3472-9ccf-4091-9bde-971d33174470)

**Comment by DanTch** [comment]
Delete files

**Reply by QuartzWarrior** [reply]
What files? I don't have any codespaces made.

**Comment by DanTch** [comment]
Message to support
